response_operations_ui/forms.py

```diff
@@ -75,16 +75,22 @@ class RuSearchForm(FlaskForm):
 
 
 class EditContactDetailsForm(FlaskForm):
+    def strip_whitespace(s):
+        if isinstance(s, str):
+            s = s.strip()
```

Comment:
Looks good, though I have a question.  Was this only an issue of space before and after?  Was there an issue of space in the middle?  
What happens if I provide `test email@example.com` does it fail?  I'm fairly sure it should

Reply:
That's right, if a user enters a leading or trailing space the code should silently strip the spaces and continue. But if there is a space inside the email address it should fail validation.